server/middlewares/errors.js

```diff
@@ -0,0 +1,28 @@
+/**
+  *  Error Handling Middleware
+  * @param {object} err
+  * @param {object} req
+  * @param {object} res
+  * @param {function} next
+  * @returns {object} Containing Error Message
+  */
+const errorHandler = (err, req, res, next) => {
+  res.status(500).json({
+    status: res.statusCode,
```

Comment:
Great work here Ayodeji! I like how you've abstracted the error handling logic. Could you please check however that the response specification is according to what was agreed on for the [JSend](https://github.com/omniti-labs/jsendhttps://github.com/omniti-labs/jsend) specification for server errors? 

Once again, this is fantastic stuff!

Reply:
Thanks for pointing it out, Tolu.  Necessary adjustments will be made, including updating the swagger doc. Thanks.